chat-server/src/routes/conversations/addMessageToConversation.ts

```diff
@@ -46,9 +50,9 @@ export interface AddMessageToConversationRouteParams {
   store: EmbeddedContentStore;
   conversations: ConversationsServiceInterface;
   embed: EmbedFunc;
-  llm: LlmProvider<OpenAiStreamingResponse, OpenAiAwaitedResponse>;
+  llm: Llm<OpenAiStreamingResponse, OpenAiAwaitedResponse>;
   dataStreamer: DataStreamerServiceInterface;
-  findNearestNeighborsOptions?: Partial<FindNearestNeighborsOptions>;
+  findNearestNeighborsOptions: FindNearestNeighborsOptions;
```

Comment:
Why this change?

Even if we require findNearestNeighborsOptions, which I'm not sure we should, we probably still want to allow defaults for most of the fields (hence `Partial<...>`)?

Reply:
i don't recall why i did this. probably part of a debugging cycle. my instinct is also that it shouldn't be changed from what it was.

Reply:
fixed. had to do change in a few other places too.